components/root_layout/root_layout_body.vue

```diff
@@ -91,6 +106,10 @@ export default {
         { 'root-layout__body--invert': this.sidebarPosition === ROOT_LAYOUT_SIDEBAR_POSITIONS.RIGHT },
       ];
     },
+
+    mainHeight () {
+      return `calc(100vh - (${this.headerHeight} + ${this.footerHeight}))`;
```

Comment:
We need to consider the case in small viewport (`< 512px`) where the sidebar is on top of the content reducing the viewport size so the sidebar height should be added to this calc.

Reply:
Was a bit difficult to determine whether the sidebar was above or below the footer due to it being pure css, however I managed to do it with a resize observer and offsetTop.